cypress-io/cypress
# chore: specify TS 5.x support

- Closes #26204

### Additional details


Specify Cypress CT works with TypeScript 5.x.

### Steps to test

You could make a project using CT and TS 5.x There's zillions of permutations (Next, Vue, Angular...). No doubt some specific combinations might not work, but in general, none of the [breaking changes](https://devblogs.microsoft.com/typescript/announcing-typescript-5-0/#breaking-changes-and-deprecations) impact us from what I can see.

### How has the user experience changed?


Now when setting up CT, users with TS 5.x installed won't feel like it's not supported when going through the onboarding workflow, which is currently the case: #26204. This makes for a bad onboarding and first 5 minutes experience, which is essential when adopting a new tool.

### PR Tasks
